pom.xml

```diff
@@ -18,24 +18,24 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>co.cask.cdap</groupId>
+  <groupId>io.cdap.plugin</groupId>
   <artifactId>state-restore-action</artifactId>
-  <version>1.1.0</version>
+  <version>1.2.0-SNAPSHOT</version>
   <packaging>jar</packaging>
 
   <name>State Restore Action</name>
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdap.version>4.1.0</cdap.version>
-    <hydrator.version>1.6.0</hydrator.version>
+    <cdap.version>6.1.0-SNAPSHOT</cdap.version>
+    <hydrator.version>2.2.0</hydrator.version>
     <guava.version>19.0</guava.version>
     <!-- properties for script build step that creates the config files for the artifacts -->
```

Comment:
can you also update the pom to use the cdap-maven-plugin instead of the big javascript antrun plugin, and then remove these unneeded properties